app-misc/tmux/patches/tmux-3.1c.patchset

```diff
@@ -176,22 +188,70 @@ index 4d54353..34f1a23 100644
  		tio.c_iflag |= IGNBRK;
  		tio.c_oflag &= ~(OPOST|ONLCR|OCRNL|ONLRET);
  		tio.c_lflag &= ~(IEXTEN|ICANON|ECHO|ECHOE|ECHONL|ECHOCTL|
-diff --git a/configure.ac b/configure.ac
-index 5cce1d9..32ec9c0 100644
---- a/configure.ac
-+++ b/configure.ac
-@@ -284,9 +284,9 @@ AC_TRY_LINK(
- if test "x$found_b64_ntop" = xno; then
- 	AC_MSG_RESULT(no)
+-- 
+2.28.0
+
+
+From 75c201236ccefd294162e9535384f68dd93700b5 Mon Sep 17 00:00:00 2001
+From: Zach Dykstra <dykstra.zachary@gmail.com>
+Date: Wed, 30 Dec 2020 00:30:38 -0600
+Subject: Update search locations for tmux.conf
+
+
+diff --git a/Makefile.am b/Makefile.am
+index 082a467..a867aa2 100644
+--- a/Makefile.am
++++ b/Makefile.am
+@@ -13,7 +13,7 @@ dist_EXTRA_tmux_SOURCES = compat/*.[ch]
+ # Preprocessor flags.
+ AM_CPPFLAGS += @XOPEN_DEFINES@ \
+ 	-DTMUX_VERSION="\"@VERSION@\"" \
+-	-DTMUX_CONF="\"$(sysconfdir)/tmux.conf:~/.tmux.conf:~/.config/tmux/tmux.conf\""
++	-DTMUX_CONF="\"$(shell finddir B_USER_SETTINGS_DIRECTORY)/tmux.conf:~/.tmux.conf:~/.config/tmux/tmux.conf\""
  
--	AC_MSG_CHECKING(for b64_ntop with -lresolv)
-+	AC_MSG_CHECKING(for b64_ntop with -lnetwork)
- 	OLD_LIBS="$LIBS"
--	LIBS="$LIBS -lresolv"
-+	LIBS="$LIBS -lnetwork"
- 	AC_TRY_LINK(
- 		[
- 			#include <sys/types.h>
+ # Additional object files.
+ LDADD = $(LIBOBJS)
+@@ -198,7 +198,7 @@ endif
+ # Install tmux.1 in the right format.
+ install-exec-hook:
+ 	if test x@MANFORMAT@ = xmdoc; then \
+-		sed -e "s|@SYSCONFDIR@|$(sysconfdir)|g" $(srcdir)/tmux.1 \
++		sed -e "s|@SYSCONFDIR@|$(shell finddir B_USER_SETTINGS_DIRECTORY)|g" $(srcdir)/tmux.1 \
```

Comment:
Why is this needed?

Reply:
It's needed so that the man page showed the correct paths based on the other changes included in the patch.